cmake/modules/RootBuildOptions.cmake

```diff
@@ -154,7 +154,6 @@ ROOT_BUILD_OPTION(rfio OFF "RFIO support, requires libshift from CASTOR >= 1.5.2
 ROOT_BUILD_OPTION(roofit ON "Build the libRooFit advanced fitting package")
 ROOT_BUILD_OPTION(root7 OFF "Build the ROOT 7 interface prototype, requires >= cxx14")
 ROOT_BUILD_OPTION(rpath OFF "Set run-time library load path on executables and shared libraries (at installation area)")
-ROOT_BUILD_OPTION(ruby OFF "Ruby ROOT bindings, requires ruby >= 1.8")
```

Comment:
Same as for chirp, could the code be removed together with the build system parts?
Commit message ends with a dot, is that intentional?

Reply:
I wasn't going to remove ruby just yet, "only" disable it in the build system. I want to make sure that we know wether "broken since years, nobody is using it" actually holds.